### ISSUE TITLE
Unmaintained dependency iconv_hook-devel / apache2-mod_encoding

### ISSUE THREAD

**Issue by sebix** [folkertvanheusden/rsstail#4]
Hi there,

rsstail uses depends on libiconv hook which is provided by apache2's mod_encoding. The latter is not maintained anymore,  no the new version was released 2002 and now fails to build with gcc14.

This is just for your information. We need to drop rsstail because of that in openSUSE now.

Best regards

**Comment by folkertvanheusden** [comment]
Thank you for your message.

I cannot reproduce the problem? (on ubuntu though)
I mean it still compiles.

**Comment by sebix** [comment]
Do you have an Ubuntu installation with gcc 14? According to https://packages.ubuntu.com/search?suite=oracular&searchon=names&keywords=gcc the also the latest (and still unreleased) development version uses gcc 13

**Comment by folkertvanheusden** [comment]
Hi,

The latest ubuntu does have g++14 indeed.

***@***.*** ~/Projects/rsstail (master)$ dpkg --list | grep gcc-14
ii  gcc-14
14-20240412-0ubuntu1                        amd64        GNU C compiler
ii  gcc-14-base:amd64
 14-20240412-0ubuntu1                        amd64        GCC, the GNU
Compiler Collection (base package)
ii  gcc-14-base:i386
14-20240412-0ubuntu1                        i386         GCC, the GNU
Compiler Collection (base package)
ii  gcc-14-cross-base
 14-20240412-0ubuntu1cross1                  all          GCC, the GNU
Compiler Collection (library base package)
ii  gcc-14-x86-64-linux-gnu
 14-20240412-0ubuntu1                        amd64        GNU C compiler
for the x86_64-linux-gnu architecture
ii  libgcc-14-dev:amd64
 14-20240412-0ubuntu1                        amd64        GCC support
library (development files)

Found that iconv_hook is no longer required. I've updated the makefile
accordingly.

Thanks for reporting.
